src/overlays/actors/ovl_En_Ex_Ruppy/z_en_ex_ruppy.c

```diff
@@ -21,29 +34,345 @@ const ActorInit En_Ex_Ruppy_InitVars = {
     (ActorFunc)EnExRuppy_Update,
     (ActorFunc)EnExRuppy_Draw,
 };
-*/
+
+Vec3f D_80A0B358[] = {{0.0f, 0.1f, 0.0f}, {0.0f, 0.0f, 0.0f}};
+Vec3f D_80A0B370[] = {{0.0f, 0.01f, 0.0f},{0.0f, 0.0f, 0.0f}};
+Vec3f D_80A0B388 = {0.0f, 0.1f, 0.0f};
+Vec3f D_80A0B394 = {0.0f, 0.0f, 0.0f};
+static Vec3f sPoint1Vec = {0x0, 0.1f,0x0};
+static Vec3f sZeroVector = {0.0f, 0.0f, 0.0f};
+s32 D_80A0B3B8[] = {0x04042140, 0x04042160, 0x04042180, 0x040421C0, 0x040421A0, 0x00000000};
+
```

Comment:
beacuse of my suggestion above you will need to move this down. you can put it right above the function its used.
also, UNK_PTR